pkg/model/process_energy.go

```diff
@@ -304,3 +283,29 @@ func addEstimatedEnergy(processIDList []uint64, processesMetrics map[uint64]*sta
 		}
 	}
 }
+
+// addGPUEnergy estimates the GPU power consumption
+func addGPUEnergy(processIDList []uint64, processesMetrics map[uint64]*stats.ProcessStats, nodeMetrics *stats.NodeStats) {
+	// estimate the associated power comsumption of GPU for each process
+	if !gpu.IsGPUCollectionSupported() {
+		return
+	}
+	for gpuID := range gpu.GetGpus() {
+		gpuName := fmt.Sprintf("%s%v", utils.GenericGPUID, gpuID)
+		numberOfProcesses := len(processIDList)
+		// calculate the gpu's processes energy consumption for each gpu
+		processIdlePower := nodeMetrics.EnergyUsage[config.IdleEnergyInGPU].GetDeltaValuesByKey(gpuID) / uint64(numberOfProcesses)
```

Comment:
The idle power is a part of the node and should be divided among all processes, regardless of GPU usage.

So the process idle power should not be relative to only one GPU.

Reply:
So, we just need to get here the Sum of the Idle power of all GPUs.
`EnergyUsage[config.IdleEnergyInPkg].SumAllDeltaValues()`

But note that the list of GPU in `gpu.GetGpus()`, should be only the Physical GPUs. If it also has the MIG slices, the `SumAllDeltaValues` will double count the idle power of all MIG slices, as it has the GPU power.
The PR #1299 should help with this since it keeps only the  Physical GPUs in the `gpu.GetGpus()`.